### PR TITLE
Update version to 2020.9.0

### PR DESCRIPTION
**Pull request by nsorros**
Description
---

Checklist
---

- [ ] Added link to Github issue or Trello card
- [ ] Added tests

### CODE REVIEW

**Comment by aCampello** [comment]
Can we wait until we merge https://github.com/wellcometrust/WellcomeML/pull/109 ?

**Comment by nsorros** [comment]
> Can we wait until we merge #109 ?

Ok, let me review it as well.

**Review by aCampello — APPROVED** [review]
Nice. Ready to generate a new release.

Feel free to copy paste them output of the `make` commands for a new release in case they fail.

**Pull request merged**
